egs/wsj/s5/tensor_2048_relu.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2016    Vincent Renkens
```

Comment:
Also, is this the top-level example script?  I'm surprised there isn't a top-level shell script, that would make sure everything is in place... e.g. I saw something about needing un-compressed features.  Did you have to modify any of the checked-in shell scripts or run anything separate, to ensure the features on disk were un-compressed?